moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -0,0 +1,260 @@
+const OnChangePlugin = ({ onChange }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    return editor.registerUpdateListener(({ editorState }) => {
+      editorState.read(() => {
+        const isEditorEmpty = $isRootTextContentEmpty();
+        const htmlContent = isEditorEmpty ? null : $generateHtmlFromNodes(editor, null);
+        onChange(htmlContent);
+      });
+    });
+  });
+  return null;
+}
+
+// On successful save, clear and refocus the editor
+const OnSavePlugin = ({ noteAddSuccess }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    noteAddSuccess && editor.update(() => {
+      const root = $getRoot();
+      root.clear();
+    });
+    editor.focus();
+  }, [noteAddSuccess, editor]);
+  return null;
+}
+
+// On edit, load the selected comment and refocus the editor
+const OnEditPlugin = ({ htmlContent, editingNote }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    const editorState = editor.getEditorState()
+    const nodeMap = editorState._nodeMap;
+    const textArray = [...nodeMap.entries()][2];
+    const isTextInEditor = !!textArray;
+    const isTextinDatabase = !!htmlContent;
+    // If user clicks edit and there is a stored note and a clear editor,
+    // populate editor with stored note
+    editingNote && !isTextInEditor && isTextinDatabase &&
+      editor.update(() => {
+        $getRoot()
+          .getChildren()
+          .forEach((n) => n.remove());
+        const parser = new DOMParser();
+        const dom = parser.parseFromString(htmlContent, 'text/html');
+        const nodes = $generateNodesFromDOM(editor, dom);
+        nodes.forEach((node) => {
+          $getRoot().append(node);
+        });
+      });
+    editor.focus();
+  }, [editingNote, htmlContent, editor]);
+  return null;
+}
+
+const NoteInput = ({
+  noteText,
+  setNoteText,
+  newNoteType,
+  setNewNoteType,
+  editingNoteType,
+  setEditingNoteType,
+  editingNote,
+  noteAddLoading,
+  noteAddSuccess,
+  submitNewNote,
+  submitEditNote,
+  cancelNoteEdit,
+  isStatusEditModal,
+}) => {
+  const classes = useStyles();
+
+  const initialConfig = {
+    namespace: 'MyEditor',
+    theme: PlaygroundEditorTheme,
+    nodes: [
+      LinkNode,
+      ListNode,
+      ListItemNode
+    ],
+    onError,
+  };
+
+  const onChange = (htmlContent) => {
+    setNoteText(htmlContent);
+  };
+
+  return (
+    <Container>
+      <Grid container direction="column" spacing={1}>
+        <Grid item xs={12} sm={12}>
+          <LexicalComposer initialConfig={initialConfig} >
+            <ToolbarPlugin noteAddSuccess={noteAddSuccess} classes={classes} />
+            <Box className={classes.editorWrapper} pt={2}>
+              <RichTextPlugin
+                contentEditable={<ContentEditable className={classes.contentEditable} />}
+                placeholder={<div className={classes.placeholder}>Enter some text...</div>}
```

Comment:
Could you pass `null` for this placeholder? We don't have any placeholder text in the the current editor, and we haven't had reports of it being confusing (yet 🙃).